Kickstarter-iOS/Views/Controllers/BackerDashboardViewController.swift

```diff
@@ -0,0 +1,367 @@
+  private weak var savedProjectsViewController: ProfileProjectsViewController!
+
+  fileprivate let viewModel: BackerDashboardViewModelType = BackerDashboardViewModel()
+
+  private var isCollapsed = false
+
+  internal static func instantiate() -> BackerDashboardViewController {
+    return Storyboard.BackerDashboard.instantiate(BackerDashboardViewController.self)
+  }
+
+  internal override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.backedProjectsViewController = self.childViewControllers
+      .flatMap { $0 as? ProfileProjectsViewController }.first
+
+    self.backedProjectsViewController.delegate = self
+
+    self.savedProjectsViewController = self.childViewControllers
+      .flatMap { $0 as? ProfileProjectsViewController }.last
+
+    self.savedProjectsViewController.delegate = self
+
+    _ = self.backedMenuButton
+      |> UIButton.lens.targets .~ [(self, action: #selector(backedButtonTapped), .touchUpInside)]
+
+    _ = self.savedMenuButton
+      |> UIButton.lens.targets .~ [(self, action: #selector(savedButtonTapped), .touchUpInside)]
+
+    _ = self.messagesButtonItem
+      |> UIBarButtonItem.lens.targetAction .~ (self, #selector(messagesButtonTapped))
+
+    _ = self.settingsButtonItem
+      |> UIBarButtonItem.lens.targetAction .~ (self, #selector(settingsButtonTapped))
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  internal override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+
+    self.viewModel.inputs.viewWillAppear(animated)
+  }
+
+  // swiftlint:disable:next function_body_length
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.avatarImageView.rac.imageUrl = self.viewModel.outputs.avatarURL
+    self.backedContainerView.rac.hidden = self.viewModel.outputs.backedProjectsAreHidden
+    self.savedContainerView.rac.hidden = self.viewModel.outputs.savedProjectsAreHidden
+    self.backerNameLabel.rac.text = self.viewModel.outputs.backerNameText
+    self.backerLocationLabel.rac.text = self.viewModel.outputs.backerLocationText
+    self.sortBar.rac.hidden = self.viewModel.outputs.sortBarIsHidden
+
+    self.viewModel.outputs.backedButtonTitleText
+      .observeForUI()
+      .observeValues { [weak self] string in
+        guard let _self = self else { return }
+        _self.setAttributedTitles(for: _self.backedMenuButton, with: string)
+    }
+
+    self.viewModel.outputs.configureBackedProjectsController
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.backedProjectsViewController.configureWith(projectsType: $0)
+    }
+
+    self.viewModel.outputs.configureSavedProjectsController
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.savedProjectsViewController.configureWith(projectsType: $0)
+    }
+
+    self.viewModel.outputs.embeddedViewTopConstraintConstant
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.embeddedViewsTopLayoutConstraint.constant = $0
+    }
```

Comment:
we even have a reactive extension for this! `self.embeddedViewsTopLayoutConstraint.rac.constant = ...`